plugins/modules/site_workflow_manager.py

```diff
@@ -482,29 +575,140 @@ def site_exists(self):
           'get_site' function in the 'sites' family. It utilizes the
           'site_name' parameter from the 'want' attribute to identify the site.
         """
-
         site_exists = False
         current_site = {}
         response = None
+
+        if self.get_ccc_version_as_integer() <= self.version_2_3_5_3:
+            site_name = self.want.get("site_name")
+            response = self.get_site(site_name)
+
+            if not response:
+                self.log("No response received from 'get_site' API for site: {0}".format(site_name), "ERROR")
+                return (site_exists, current_site)
+
+            if response:
```

Comment:
if response: 
is not required. You have already checked if not reponse in line 586.. Hence at this point the response is true.

Reply:
Removed